en/includes/guides/organization-management/organization-applications.md

```diff
@@ -6,7 +6,8 @@ from the organizations.
 These applications can be created under the following conditions.
 
 - Application's protocol is OAuth2.
-- Only client credentials, password and refresh grant types can be used.
+- Only authorization code, client credentials, password and refresh grant types can be used.
```

Comment:
Since `authorization code` support was not in 7.1.0 and 7.2.0 right?
Content should handle this conditionally/ with templates
Fix other places as well